config/prod/htan-dcc-chop.yaml

```diff
@@ -0,0 +1,29 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html) that will
+# work with the s3-synapse-sync lambda: https://github.com/Sage-Bionetworks/s3-synapse-sync
+template_path: "remote/s3-synapse-sync-bucket.j2"
+stack_name: "htan-dcc-chop"
+stack_tags:
+  Department: "CompOnc"
+  Project: "HTAN"
+  OwnerEmail: "xengie.doan@sagebase.org"
+hooks:
+  before_launch:
+    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/s3-synapse-sync/master/s3-synapse-sync-bucket.j2 --create-dirs -o templates/remote/s3-synapse-sync-bucket.j2"
+dependencies:
+  - "prod/s3-synapse-sync.yaml"
+parameters:
+  BucketName: "htan-dcc-chop"  # must match bucket name in data/s3-synapse-sync-bucket-vars.yaml
+  SynapseIDs:
+    - "3391844" #HTAN DCC
+    - "3413795" #service acct
```

Comment:
would be nice to update the other htan buckets with these comments.

Reply:
I can update with comments

Reply:
how about a PR for this update?